Source/ACE.Entity/AceObject.cs

```diff
@@ -1220,6 +1220,12 @@ public string DisplayName
             set { SetBoolProperty(PropertyBool.RequiresBackpackSlot, value); }
         }
 
+        public bool UseBackpackSlot
```

Comment:
Feels like this shouldn't be in AceObject and instead in WorldObject. I follow why you put it here though. GameEventPlayerDescription reads things as AceObject and not WorldObject. 

Maybe slap a few fixme/todos to correct the anomalous property/object usage so when someone gets around to bringing GameEventPlayerDescription in line with expectations this oddity gets corrected?